packages/blip/app/components/chart/trendssubnav.js

```diff
@@ -33,16 +33,13 @@ class DaysGroup extends React.Component {
   }
 
   render() {
-    var groupClass = cx({
-      daysGroup: true,
```

Comment:
Shouldn't `.daysGroup` be deleted from `packages/blip/app/components/chart/trendssubnav.less`?

Reply:
You're right, good catch 👀